Clarify Ledger Live information regarding public address privacy (#583)
zkSNACKs/WasabiDoc: docs/FAQ/FAQ-UseWasabi.md

:::

:::details

> ### Does Ledger Live send my public addresses to a third party server?

Reviewer: ```suggestion
### Does Ledger Live send my public keys and addresses to a third party server?
```

to stay consistent with the rest of the doc, we always call it `public key` or `address`.

Only if you add your accounts in the app, but not if you update your device firmware or install apps. 

> When using the Ledger Live software wallet to manage your coins, you send all of your used, and 20 unused public addresses to Ledger's nodes. [Source](https://support.ledger.com/hc/en-us/articles/360010892360)

Reviewer 2: ```suggestion
When using the Ledger Live software wallet to manage your coins, you send all of your used, and 20 unused addresses to Ledger's nodes. [Source](https://support.ledger.com/hc/en-us/articles/360010892360)
```

> Ledger says it does not use this information to track its users' coins and transactions, but as a user you cannot verify this.

Reviewer: ```suggestion
With this information they know exactly how many bitcoins you have and in what transactions you spend them.
```

You do track the coins, that's what this is for, so that the user knows how many coins he has...
The question would be how long you store the addresses, if you log the IP address, timing data, or other meta data.

Author: We do not keep logs of the API calls to the nodes, there's no record of IP addresses and coin amounts during normal node operation. So saying that "they know exactly how many bitcoins you have" is a bit of an overstatement.
Of course, these things cannot be verified, hence the recommendation to run one's own node or use a wallet like Wasabi for better privacy guarantees.

Reviewer: I try to write these docs out of the perspective of what is sent from the users' computer. Not necessarily what happens with the data on the computer of someone else, as this cannot be verified in any way. 

And what happens is that the user sends his addresses to your server. The privacy leak is done by then - the worst case has happened.

Author: I've added more nuance in my latest commit. Please review and let me know if it's a good compromise for you :)

Your extended public key, however, is not shared with Ledger's node, but rather stored encrypted on your local machine. [Source](https://support.ledger.com/hc/en-us/articles/360011069619)

> To avoid this privacy leak, you can use a Ledger hardware wallet in combination with Wasabi as a software interface, and because Wasabi does not leak your public addresses, your transaction history is not shared with anyone.

Reviewer 2: ```suggestion
To avoid this privacy leak, you can use a Ledger hardware wallet in combination with Wasabi as a software interface, and because Wasabi does not leak your addresses, your transaction history is not shared with anyone.
```

:::details
### Does Ledger Live send my public keys and addresses to a third party server?

Only if you add your accounts in the app, but not if you update your device firmware or install apps. 
When using the Ledger Live software wallet to manage your coins, you send all of your used, and 20 unused addresses to Ledger's nodes. [Source](https://support.ledger.com/hc/en-us/articles/360010892360)
Ledger says it does not use this information to track its users' coins and transactions, but as a user you cannot verify this.

> Your extended public key, however, is not shared with Ledger's node, but rather stored encrypted on your local machine. [Source](https://support.ledger.com/hc/en-us/articles/360011069619)

Reviewer 3: ```suggestion
Your extended public key, however, is not shared with Ledger's node, but rather stored encrypted on your local machine.
Read more [here](https://support.ledger.com/hc/en-us/articles/360011069619).
```

Only if you add your accounts in the app, but not if you update your device firmware or install apps. 
When using the Ledger Live software wallet to manage your coins, you send all of your used, and 20 unused addresses to Ledger's nodes. [Source](https://support.ledger.com/hc/en-us/articles/360010892360)

Your extended public key, however, is not shared with Ledger's node, but rather stored encrypted on your local machine.
Read more [here](https://support.ledger.com/hc/en-us/articles/360011069619).

> Though Ledger says it does not use this information to try to identify how much bitcoin you have and what your transactions you make, Ledger could potentially do so.

Reviewer: But this is wrong... you clearly use the addresses to identify 
- how many bitcoins are in the wallet, you show the total balance.
- what transactions are made, you show the history of transactions.

Author: An API that answers queries without systematically logging IP addresses + addresses is not the same as actively trying to identify you as a person. 

You seem to be assuming that there is:
1. systematic collection of data
2. combination of data
3. analysis of data (e.g. clustering, filtering, etc.)

There is no legal ground to do any of the above since providing a non-custodial, light-weight wallet service does not require any knowledge of which entity is using it (refer to GPDR). It's a different case for custodial services, that are obliged to identify users (KYC/AML). 

I keep repeating that users have no guarantees that 1, 2 and 3 are not happening so, therefore, they should run their own node to have stronger privacy guarantees. Indeed, they could also file a GPDR complaint if they suspect it is happening and without legal ground (e.g. explicit consent), with severe penalties for the company when the authorities confirm so.

Reviewer: Yes, the issue is not about KYCing the users. But it is about clustering their coins. As soon as the user sends his addresses to any third party server, there is huge risk of these addresses being clustered.
